app/api-v1/api-doc.js

```diff
@@ -95,6 +95,11 @@ const apiDoc = {
           },
         },
       },
+      OnChainAddress: {
+        type: 'string',
+        description: 'A literal which will be represented as on-chain metadata',
+        maxLength: 55,
+      },
```

Comment:
This is unused and should likely be removed

Reply:
removed, to be fair not sure how it got there 🤔